### ISSUE TITLE
Events with no UID

### ISSUE THREAD

**Issue by adrienfr** [u01jmg3/ics-parser#172]
- PHP Version: 7.0.22
- PHP date.timezone: UTC
- ICS Parser Version: 2.1.2
- Linux

# Description of the Issue:

I think there is an issue with events which don't have `UID` attribute, got this error:

```
Notice: Undefined index: UID
```

https://github.com/u01jmg3/ics-parser/blob/4fec38e3e04b47d49125c6dd217770384e1f88ac/src/ICal/ICal.php#L856

**Comment by u01jmg3** [comment]
> _The `UID` property MUST be specified in the "VEVENT", "VTODO", "VJOURNAL" or "VFREEBUSY" calendar components._

- https://www.kanzaki.com/docs/ical/uid.html

**Comment by adrienfr** [comment]
True but since this check is done juste before:
```
$checks[] = isset($event['UID']);
```
Events with no `UID` should be skipped or exception thrown no?

**Comment by u01jmg3** [comment]
Only poor iCal software omits a `UID` but feel free to PR.

**Comment by u01jmg3** [comment]
Closed by #173

**Comment by adrienfr** [comment]
Thanks @Big-Shark & @u01jmg3 !
